egs/tts/VALLE/exp_config.json

```diff
@@ -25,5 +25,8 @@
         "batch_size": 4,
         "train_stage": 1, // 0: train all modules, For VALL_E, support 1: AR Decoder 2: NAR Decoder(s)
         "max_epoch": 20, // "Number of epochs to train."
+        "use_dynamic_batchsize": false, // If use dynamic batch size
```

Comment:
"use_dynamic_batchsize" is used in tts_trainer.py, indicating that it could impact other TTS models. So "use_dynamic_batchsize" should be set in config/tts.json.

Besides, the settings of "use_dynamic_batchsize": true and  "optimizer": "AdamW" could be set in config/valle.json as valle's default settings.